standards/SPIFFE_Workload_API.md

```diff
@@ -203,7 +194,7 @@ The `X509SVIDRequest` request message is currently empty and is a placeholder fo
 
 The `X509SVIDResponse` response consists of a mandatory `svids` field, which MUST contain one or more `X509SVID` messages (one for each identity granted to the client). The `crl` and `federated_bundles` fields are optional. 
 
-All fields in the `X509SVID` message are mandatory.
+All fields in the `X509SVID` message are mandatory, with the exception of the `hint` field. When the `hint` field is set, its value MUST be unique amongst the set of returned SVIDs in the `X509SVIDResponse` message.
```

Comment:
Here is the suggestion we discussed on the last SIG-Spec call. There is some uncertainty around whether or not the uniqueness requirement would do more harm than good. The argument for uniqueness is "we haven't solved the disambiguation problem if the hints aren't unique". The argument against is "it seems like there may be cases where we would want duplicated hint values, and it's not clear if the hint in its current shape is wholly sufficient for all use cases" ... please correct me if this summary is off :)

In the interest of progress, we decided to have a look at the option below. By specifying client behavior in the presence of duplicate `hint` values, we think we can de-risk the decision because it will be safer to relax the requirement in the future if necessary.

```suggestion
All fields in the `X509SVID` message are mandatory, with the exception of the `hint` field. When the `hint` field is set (i.e. non-empty), SPIFFE Workload API servers MUST ensure its value is unique amongst the set of returned SVIDs in any given `X509SVIDResponse` message. In the event that a SPIFFE Workload API client encounters more than one `X509SVID` message with the same `hint` value set, then the first message in the list SHOULD be considered the winner.
```

Reply:
SHOULD be selected